learn/cookbooks/gcp.md

```diff
@@ -80,7 +78,7 @@ Configuring your Meilisearch instance in a production environment is not just st
 
 ### 1. Make your domain name point to your instance IP
 
-If you want to use a custom domain name (or sub-domain), add an `A record` in your domain name provider account. Otherwise, you can skip this step.
+If you want to use a custom domain name (or sub-domain), add a record to your domain name provider account. Otherwise, you can skip this step.
```

Comment:
⚠️ Caution, an "A record" is an actual thing! ⚠️ 

So you'll want to revert this change. [Here's a link](https://docs.gandi.net/en/domain_names/faq/record_types/a_record.html) where you can read a little more about it.